src/zc/buildout/easy_install.py

```diff
@@ -560,6 +625,9 @@ def _get_dist(self, requirement, ws):
 
             # Retrieve the dist:
             if avail is None:
+                # TODO I don't think the next line is useful.  When we get
+                # here, I expect that 'obtain' was already called and has
+                # returned None, which is why we are here.
```

Comment:
Is there any change in behaviour of the tests when omitting the next line? Or is this code path untested?

Reply:
I tried, and the extra `_obtain` call is safe to remove.  I have removed it now, and added some explanatory comments in this method.